cranelift/filetests/filetests/runtests/fdemote.clif

```diff
@@ -73,21 +73,3 @@ block0(v0: f64):
 ; run: %fdemote_is_nan(-sNaN:0x1) == 1
 ; run: %fdemote_is_nan(+sNaN:0x4000000000001) == 1
 ; run: %fdemote_is_nan(-sNaN:0x4000000000001) == 1
-
```

Comment:
Most of the changes from the unit tests are very mechanical. I kept as many test cases in the existing file, and split out 32 and 64 bit-specific unit tests to `fdemote_32.clif` and `fdemote_64.clif` respectively.

Most often the change is the number in the `stack_addr.<size>` function I think... this could definitely do with a bit of scrutiny as I'm pretty inexperienced with these very low level constructs.